docs/docker-compose.yml

```diff
@@ -1,6 +1,13 @@
 version: '2'
 
 services:
+    smr-docs:
+        image: nginx:alpine
+        volumes:
+            - ./api/html:/usr/share/nginx/html:ro
+        ports:
+            - "8081:80"
```

Comment:
8081 is the external ip right?

Reply:
That's right. I hard-coded the external port for now because I don't suspect anyone but me will be using this. But if you want it to be a docker-compose environment variable, I can do that.